quic/s2n-quic-qns/src/client/perf.rs

```diff
@@ -25,6 +24,10 @@ pub struct Perf {
     #[structopt(long)]
     connections: Option<usize>,
 
+    /// Defines the number of concurrent connections to open at any given time
```

Comment:
Can you add a comment on the `connections` option above to make it more clear how it differs from `concurrency`?